src/common.h

```diff
@@ -106,7 +106,7 @@ static inline int msg_sendto(int fd, uint8_t *buf, int nbytes,
         addrlen = 0;
     }
 
-    while (nbytes) {
+    while (nbytes && !g_b_exit) {
```

Comment:
The question is why it can't send ...

Reply:
> The question is why it can't send ... Patching with g_b_exit is correct on one hand but can hide issues on the other hand

Should not be covered by
```
        } else if (ret < 0 && (errno == EINTR)) {
            /* A signal occurred.
             */
            errno = 0;
            break;
```